docs/components_and_bundles/bundles/SyliusOrderBundle/processors.rst

```diff
@@ -47,10 +47,10 @@ In a controller:
     <?php
 
     // Fetch order from DB
-    $order = ...;
+    $order = $this->container->get('sylius.repository.order')->find('$id');
 
     // Get the processor from the container or inject the service
-    $orderProcessor = ...;
+    $orderProcessor = $this->container->get('sylius.order_processing.shipping_charges_processor');
```

Comment:
Should be `sylius.order_processing.order_processor`

Reply:
Well I just took one of the processors as example.

Reply:
I thought so, but this part is about the composite order processor.

Reply:
I'm voting for `sylius.order_processing.order_processor` here